src/Sulu/Bundle/AdminBundle/Resources/js/components/Checkbox/Checkbox.js

```diff
@@ -0,0 +1,54 @@
+// @flow
+import React from 'react';
+import type {ElementRef} from 'react';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import checkboxStyles from './checkbox.scss';
+
+type Props = {
+    checked: boolean,
```

Comment:
I think it does. I don't even know right now how to implement that in a good way. Would you have a `value` and a `checked` property that are mutual exclusive? Or are we getting rid of the `checked` property and only use a nullable `value`?

Regarding the list view: In that case the list view could take care of the values, couldn't it?